### PR TITLE
Added Contributing to README.md

### PR DESCRIPTION
**Pull request by yashlamba**
Didn't add the gitter badge in this as did already in #67. References #60.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,18 +60,19 @@ To start using `dffml` for machine learning with a few CLI commands see
...
-Making you're next machine learning project a breeze to write!
-
-- Features
-  - The [new feature tutorial](docs/tutorial/FEATURE.md) will walk you through
-    how to write a new DFFML feature to generate data for a dataset.
-- Models
-  - The [new model tutorial](docs/tutorial/MODEL.md) will walk you through how
-    to wrap your favorite framework or a custom implementation in the DFFML
-    library's model API.
+## Contributing
```
Comment (pdxjohnny) [inline README.md]: Please leave the Tutorials section and add the Contributing section above it.

File: README.md [hunk 2]
```diff
@@ -60,18 +60,19 @@ To start using `dffml` for machine learning with a few CLI commands see
...
+
+DFFML is meant to be a community driven application. There are various segments you can help with:
+
+- **Issues**
```
Comment (pdxjohnny) [inline README.md]: Can you remove the `**Issues**` title and keep the bullet points below it.

File: README.md [hunk 3]
```diff
@@ -60,18 +60,19 @@ To start using `dffml` for machine learning with a few CLI commands see
...
+    - Found a bug, error in the docs or have a new idea, create an issue [here](https://github.com/intel/dffml/issues/new/choose).
+    - Help us fix an [issue](https://github.com/intel/dffml/issues), refer to [HACKING](docs/HACKING.md) in the docs.
```
Comment (pdxjohnny) [inline README.md]: Can you add a bullet point asking people to read the `[CONTRIBUTING.md](CONTRIBUTING.md)` doc to learn about how to get in touch.

File: README.md [hunk 4]
```diff
@@ -60,18 +60,19 @@ To start using `dffml` for machine learning with a few CLI commands see
...
+
+- **Add new Features/Models:** Got an idea for a new feature/model, tutorials will help you write code that takes full advantage of the DFFML API. Making your next machine learning project a breeze to write!
+    - Features: The [new feature tutorial](docs/tutorial/FEATURE.md) will walk you through how to write a new DFFML feature to generate new data for a dataset.
+    - Models: The [new model tutorial](docs/tutorial/MODEL.md) will walk you through how to wrap your favorite framework or a custom implementation in the DFFML library's model API.
+
+- **Community:** Join the community. Ask doubts, share ideas, request features and what not. Join our [gitter](https://gitter.im/dffml/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link) room and say Hi!
```
Comment (pdxjohnny) [inline README.md]: Please remove `?utm_source=share-link&utm_medium=link&utm_campaign=share-link`  from the gitter link. And move this bullet to it's own section in `CONTRIBUTING.md`
Also, the word `doubts` should be `questions`

File: README.md (hunk 1 above)
Comment (yashlamba) [reply README.md]: Should I make them nested of tutorials should be separate?

**Comment by yashlamba** [comment]
@pdxjohnny, I made tutorials nested. Please Let me know if it needs to be changed.

**Pull request merged**